Makefile

```diff
@@ -70,6 +72,7 @@ release:
 	GOOS=darwin GOARCH=arm64 $(MAKE) libzstd.a
 	GOOS=darwin GOARCH=amd64 $(MAKE) libzstd.a
 	GOOS=windows GOARCH=amd64 $(MAKE) libzstd.a
+	GOOS=freebsd GOARCH=amd64 $(MAKE) libzstd.a
```

Comment:
Please update the comment on the line 62, since it looks like Zig cross-compiler supports builds for FreeBSD.